MediaBrowser.Providers/Plugins/MusicBrainz/Configuration/PluginConfiguration.cs

```diff
@@ -1,37 +1,58 @@
-#pragma warning disable CS1591
-
 using MediaBrowser.Model.Plugins;
+using MetaBrainz.MusicBrainz;
+
+namespace MediaBrowser.Providers.Plugins.MusicBrainz.Configuration;
 
-namespace MediaBrowser.Providers.Plugins.MusicBrainz
+/// <summary>
+/// MusicBrainz plugin configuration.
+/// </summary>
+public class PluginConfiguration : BasePluginConfiguration
 {
-    public class PluginConfiguration : BasePluginConfiguration
-    {
-        private string _server = Plugin.DefaultServer;
+    private const string DefaultServer = "musicbrainz.org";
+
+    private const double DefaultRateLimit = 1.0;
 
-        private long _rateLimit = Plugin.DefaultRateLimit;
+    private string _server = DefaultServer;
+
+    private double _rateLimit = DefaultRateLimit;
+
+    /// <summary>
+    /// Gets or sets the server url.
+    /// </summary>
+    public string Server
+    {
+        get => _server;
 
-        public string Server
+        set
         {
-            get => _server;
-            set => _server = value.TrimEnd('/');
+            _server = value.TrimEnd('/');
+            Query.DefaultServer = _server;
```

Comment:
Don't like this, same below

Reply:
What would be better?

Reply:
Probably using the `ConfigurationChanged` event

Reply:
Not sure if I did this correct in the last commit but that's how the OpenSubtitlesDownloader handles this.